capa/main.py

```diff
@@ -788,9 +796,15 @@ def main(argv=None):
             len([i for i in filter(lambda r: "capa/subscope-rule" not in r.meta, rules.rules.values())]),
         )
         if args.tag:
+            n_rules_all = len(rules)
             rules = rules.filter_rules_by_meta(args.tag)
-            logger.debug("selected %s rules", len(rules))
+            n_rules = len(rules)
+            diff = n_rules_all - n_rules
+            if not args.quiet:
+                for _ in tqdm.trange(n_rules, desc="loading ", unit=" rules", postfix="skipped %d rules" % diff):
+                    pass
```

Comment:
and this here doesn't seem to do anything? is it to add a line of status output for consistency?

Reply:
correct, this is to then show the "loading" of the selected rules, e.g.
```
$ capa tests/data/Practical\ Malware\ Analysis\ Lab\ 01-01.dll_ -t mutex
loading : 100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 4/4 [00:00<?, ? rules/s, skipped 635 rules]
matching: 100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 5/5 [00:00<00:00, 386.88 functions/s, skipped 3 library functions (60%)]
+------------------------+------------------------------------------------------------------------------------+
| md5                    | 290934c61de9176ad682ffdd65f0a669                                                   |
| sha1                   | a4b35de71ca20fe776dc72d12fb2886736f43c22                                           |
| sha256                 | f50e42c8dfaab649bde0398867e930b86c2a599e8db83b8260393082268f2dba                   |
| path                   | tests\data\Practical Malware Analysis Lab 01-01.dll_                               |
+------------------------+------------------------------------------------------------------------------------+

```